aws-customerprofiles-integration/aws-customerprofiles-integration.json

```diff
@@ -23,6 +23,518 @@
         "Key",
         "Value"
       ]
+    },
+    "Object": {
+      "type": "string",
+      "maxLength": 512,
+      "pattern": "\\S+",
+      "additionalProperties": false
```

Comment:
[`additionalProperties`](https://json-schema.org/understanding-json-schema/reference/object.html#additional-properties) is specifically a keyword for `object` types